### PR TITLE
Feature/hybrid smart graphs documentation

### PR DESCRIPTION
**Pull request by hkernbach**
### Scope & Purpose

WIP
Documentation for Hybrid (Disjoint) Smart Graphs

- [ ] :hankey: Bugfix (requires CHANGELOG entry)
- [ ] :pizza: New feature (requires CHANGELOG entry, feature documentation and release notes)
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification
- [ ] :book: CHANGELOG entry made

#### Backports:

- [ ] No backports required
- [ ] Backports required for: *(Please specify versions and link PRs)*

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [ ] Enterprise PR:
- [ ] GitHub issue / Jira ticket number:
- [ ] Design document: 

### Testing & Verification

*(Please pick either of the following options)*

- [ ] This change is a trivial rework / code cleanup without any test coverage.
- [ ] The behavior in this PR was *manually tested*
- [ ] This change is already covered by existing tests, such as *(please describe tests)*.
- [ ] This PR adds tests that were used to verify all changes:
  - [ ] Added new C++ **Unit tests**
  - [ ] Added new **integration tests** (e.g. in shell_server / shell_server_aql)
  - [ ] Added new **resilience tests** (only if the feature is impacted by failovers)
- [ ] There are tests in an external testing repository:
- [ ] I ensured this code runs with ASan / TSan or other static verification tools

Link to Jenkins PR run:

### Documentation

> All new features should be accompanied by corresponding documentation. 
> Bugs and features should furthermore be documented in the CHANGELOG so that
> support, end users, and other developers have a concise overview. 

- [ ] Added entry to *Release Notes* 
- [ ] Added a new section in the *Manual* 
- [ ] Added a new section in the *HTTP API* 
- [ ] Added *Swagger examples* for the HTTP API  
- [ ] Updated license information in *LICENSES-OTHER-COMPONENTS.md* for 3rd party libraries

### External contributors / CLA Note 

Please note that for legal reasons we require you to sign the [Contributor Agreement](https://www.arangodb.com/documents/cla.pdf)
before we can accept your pull requests.

### CODE REVIEW

**Comment by hkernbach** [comment]
Currently generateSwagger.sh fails.

**Comment by hkernbach** [comment]
@Simran-B would be awesome if you could help out here.

File: Documentation/DocuBlocks/Rest/Graph/general_graph_edge_definition_add_http_examples.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ it is used with exactly the same definition. It is not possible to
 store a definition "e" from "v1" to "v2" in the one graph, and "e"
 from "v2" to "v1" in the other graph.
 
+Additionally, optional options can be set (Enterprise-Only).
```
Comment (mchacki) [inline Documentation/DocuBlocks/Rest/Graph/general_graph_edge_definition_add_http_examples.md:16]:
```suggestion
Additionally, collection creation options can be set.
```

File: Documentation/DocuBlocks/Rest/Graph/general_graph_vertex_create_http_examples.md [hunk 2]
```diff
@@ -11,6 +11,15 @@ Adds a vertex to the given collection.
 @RESTURLPARAM{graph,string,required}
 The name of the graph.
 
+@RESTBODYPARAM{options,object,optional,vertex_options}
+Additional options for the vertex to add.
```
Comment (mchacki) [inline Documentation/DocuBlocks/Rest/Graph/general_graph_vertex_create_http_examples.md:15]:
```suggestion
Additional options for the vertex collection to add.
```

**Review by mchacki — CHANGES_REQUESTED** [review]
Minor corrections

**Comment by hkernbach** [comment]
Examples needs to be generated and then copied over to: https://github.com/arangodb/arangodb/pull/14590

**Pull request merged**